src/WebExternalSearchProvider.cs

```diff
@@ -177,27 +179,42 @@ public IEnumerable<Clue> BuildClues(ExecutionContext context, IExternalSearchQue
 
             var orgWebSite = resultItem.Data.GetOrganizationWebsiteMetadata(context);
 
+            var tempUri = string.Empty;
+
+            if (orgWebSite.Logo != null && orgWebSite.Logo.ToString().StartsWith("/"))
+            {
+                tempUri = orgWebSite.Logo.ToString()[1..];
+            }
+
+            if (!string.IsNullOrEmpty(tempUri) && !Uri.IsWellFormedUriString(tempUri, UriKind.Absolute))
+            {
+                orgWebSite.Logo = new Uri(orgWebSite.ResponseUri.AbsoluteUri + tempUri);
```

Comment:
You should be able to use new Uri(Uri, string) as this will handle relative and absolute for you

Reply:
updated the code, removed unnecessary checking on uri